exampleTokens/1.tokenFactory.js

```diff
@@ -23,14 +36,12 @@ console.log('\tThe decoded token is: \n', decodedJWT)
 
 console.log("\n---- verifyJWT ----")
 
-let verifyJWT = tokensFactory.tokens.verifyJWT(signedJWT, rawPublicKey)
+let verifyJWT = tokensFactory.tokens.verifyJWT(signedJWT, '04'+config.adminPubk.substr(2))
```

Comment:
We should add a comment explaining why we use the `04`. More info at https://github.com/indutny/elliptic/issues/138